### PR TITLE
Write the CA cert to a file if this information is present in the environment variables

### PR DESCRIPTION
**Pull request by rene-dekker**
Some pods not only need to present a cert, but also trust the ca for their TLS traffic.

### CODE REVIEW

**Review by xiumozhan — APPROVED** [review]

**Review by xiumozhan — APPROVED** [review]

**Review by xiumozhan — APPROVED** [review]

**Comment by marvin-tigera** [comment]
OK, I will merge the pull request when it's ready, leave the commits as is when I merge it, and leave the branch after I've merged it.

**Pull request merged**
